packages/embed-widget/src/App.tsx

```diff
@@ -63,12 +63,17 @@ const LAYOUT_SETTINGS = {
 function App(): JSX.Element {
   const [error, setError] = useState<string>();
   const [definition, setDefinition] = useState<dh.ide.VariableDefinition>();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const [sharedObject, setSharedObject] = useState<Promise<any>>();
```

Comment:
Use `unknown` instead of `any`.
```suggestion
  const [sharedObject, setSharedObject] = useState<Promise<unknown>>();
```